src/language-html/visitor-keys.js

```diff
@@ -14,6 +14,7 @@ const visitorKeys = {
   angularControlFlowBlock: ["children", "parameters"],
   angularControlFlowBlockParameters: ["children"],
   angularControlFlowBlockParameter: [],
+  angularLetDeclaration: ["name", "value"],
```

Comment:
```suggestion
  angularLetDeclaration: [],
```

`name` and `value` are not `Node`, they are stings.